test/common/stats/lazyinit_stats_speed_test.cc

```diff
@@ -0,0 +1,168 @@
+#include "envoy/upstream/upstream.h"
```

Comment:
it is a little weird to depend on 'upstream' from 'stats'.

I'm wondering if for the purposes of this test you should just make another simple struct of stats and stat-names.

Reply:
I beg to differ. 
I think the purpose of this benchmark is to show us some data, that's connected to the change contained in this PR, where we are lazyinit the traffic stats. 


Does moving this file to test/common/upstream/ make you feel better?

Reply:
Actually I already did this for you (several years ago; I had forgotten).

See forEachSampleStat in test/common/stats/stat_test_utility.h

That just starts with raw names but it won't be hard for you to generate the stat-name structures to pass to your new `create()` factory function.

Reply:
I am not following your suggestion here. 
Are you saying we can avoid creating a copy of Upstream::ClusterTrafficStats for this test purpose?

Reply:
Yes I'm not fond of the stats test depending on upstream. All you need for the speed test is a list of stat names for testing, which we already have for testing.

Reply:
Could you show me how to get a StatsStruct outside of forEachSampleStat?

Reply:
NVM, make a copy of the ClusterTrafficStats in this file now.